api/StudentApi/Properties/launchSettings.json

```diff
@@ -18,7 +18,7 @@
     "StudentApi": {
       "commandName": "Project",
       "launchBrowser": true,
-      "applicationUrl": "https://localhost:44335;http://localhost:5000",
+      "applicationUrl": "http://localhost:5000",
```

Comment:
Front end doesn't need the other port?

Reply:
Nah, just using HTTP on local - no need for HTTPS, also makes setup easier.